metadata.rb

```diff
@@ -4,7 +4,7 @@
 maintainer_email 'splunk@cerner.com'
 license          'Apache 2.0'
 description      'Installs and configures Splunk'
-long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
+long_description((Pathname.new(__FILE__).dirname + 'README.md').read)
```

Comment:
No, this is technically a method call and rubocop demanded method parentheses be used here.